src/App.js

```diff
@@ -1,63 +1,64 @@
 import React, { useState } from "react";
-import DiaryCard from "./Components/DiaryCard";
+import { useSelector, useDispatch } from 'react-redux';
+import DiaryCard from '../src/Components/DiaryCard';
 import DiaryHome from "./Components/DiaryHome";
 
+
 function App() {
 
-  const [title, setTitle] = useState("");
-  const [description, setDescription] = useState("");
+  //read input data
+  const [newTitle, setNewTitle] = useState("");
+  const [newDescription, setNewDescription] = useState("");
+
 
-  //State for show new card
-  const [card, setCard] = useState(false);
+  //Dispatch
+  const dispatch = useDispatch();
 
-  //Object Array to map cards
-  const [cards, setCards] = useState([])
+  //Subscribe
+  const cards = useSelector(state => state.cards);
 
   function onClick(e) {
     e.preventDefault();
 
-    if (title.trim().length === 0) {
+    if (newTitle.trim().length === 0) {
       console.log("Title is missing");
       return
     }
-    if (description.trim().length === 0) {
+    if (newDescription.trim().length === 0) {
       console.log("Description is missing")
       return
     }
-
     else {
-      let newCardData = { "title": title, "description": description }
-      setCards([...cards, newCardData]);
-      setCard(true);
-      setTitle("");
-      setDescription("")
+      dispatch({ type: "addNew", sendTitle: newTitle, sendDescription: newDescription, status: true });
+      setNewTitle("");
+      setNewDescription("")
     }
   }
 
   return (
     <div className="App">
 
       <DiaryHome
+        onChangeTitle={(e) => { setNewTitle(e.target.value) }}
+        onChangeDescription={(e) => { setNewDescription(e.target.value) }}
+        title={newTitle}
+        description={newDescription}
         onClick={onClick}
-        onChangeTitle={(e) => { setTitle(e.target.value) }}
-        onChangeDescription={(e) => { setDescription(e.target.value) }}
-        title={title}
-        description={description} />
-
-      {card ?
-        cards.map((cards, index) => {
+      />
+      {
+        cards.slice(1).map((cards, index) => {
```

Comment:
use empty array for redux cards state then you do not need to remove first element of that array when display.